### DISCUSSION TITLE
AutoStop immer auf true

### DISCUSSION THREAD

**Discussion by Stephan-S** [Mogli12/KeyboardSteer#744] [Previous version]
Hallo Stefan,

ich spiele mit Lenkrad und Pedalen und habe leider das Problem, dass sobald VCA installiert ist und die ShuttleControl auf dem jeweiligen Fahrzeug aktiv ist, das Fahrzeug aktiv bremst sobald ich vom Gaspedal gehe.
So ist ohne weiteres absolut kein sanftes manuelles Fahren mehr für mich möglich!

Ich habe das mal probiert in deinem 8500 Zeilen Ungetüm zu entdecken und konnte es zumindest mal auf diese Zeile runterbrechen die mir nicht ganz klar ist:
https://github.com/Mogli12/VehicleControlAddon/blob/master/FS19_VehicleControlAddon/vehicleControlAddon.lua#L5648

Hier habe ich mir einen kurzen print Befehl reingeschrieben und konnte so sehen, dass hier immer autoStop auf true gesetzt wird.

Das Getriebe ist bei den Fahrzeugen aus und auch global auf 'Off' gestellt.
Ich persönlich nutze nur die ShuttleControl Funktion und die Blickrichtungen von deinem aufwändigen Mod :D
Aber da es im MP überall genutzt wird, wäre es klasse wenn sich an dem Bremsverhalten etwas ändern könnte. Und/oder du mir kurz erklären könntest warum es sich so verhält und ich im Zweifelsfall diese eine Zeile auf 'false' ändern würde wenn es im MP sonst keine Probleme ergibt.

Grüße,
Stephan

**Comment by Mogli12** [comment]
Hi Stephan,

probier mal Version 358.

Danke, Stefan

**Comment by Stephan-S** [comment]
Ach super, das war ja eine schnelle Reaktion!
Vielen Dank. Läuft jetzt genau wie erwartet. Danke!

Ein Kleinigkeit die mir noch aufgefallen ist, die mich aber nicht weiter stört:
Die Fahrzeuge rollen während dem (automatischen) Motorstart wenn man gerade reingetabbt hat immer noch ein Stückchen, je nach Hanglage. In den 2 Sekunden kann man dann auch nicht bremsen. Ist das so gewollt?
Aber wie gesagt, kein großes Thema für mich. Ich bin jetzt happy :D

Grüße,
Stephan
